packages/bson-transpilers/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "bson-transpilers",
+  "version": "0.15.0",
+  "apiVersion": "0.0.1",
+  "productName": "BSON Transpilers",
+  "description": "Source to source compilers using ANTLR",
+  "contributors": [
+    "Anna Herlihy <herlihyap@gmail.com>",
+    "Alena Khineika <alena.khineika@mongodb.com>",
+    "Irina Shestak <irina.shestak@mongodb.com>"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "prepare": "npm run compile",
+    "precompile": "node download-antlr.js",
+    "compile": "npm run antlr4-js && npm run antlr4-py && npm run symbol-table",
+    "antlr4-js": "java -Xmx500M -cp './antlr-4.7.2-complete.jar:$CLASSPATH' org.antlr.v4.Tool  -Dlanguage=JavaScript -lib grammars -o lib/antlr -visitor -Xexact-output-dir grammars/ECMAScript.g4",
+    "antlr4-py": "java -Xmx500M -cp './antlr-4.7.2-complete.jar:$CLASSPATH' org.antlr.v4.Tool  -Dlanguage=JavaScript -lib grammars -o lib/antlr -visitor -Xexact-output-dir grammars/Python3.g4",
+    "symbol-table": "node compile-symbol-table.js",
+    "test": "mocha",
```

Comment:
We also need test-ci and check-ci in scripts

Reply:
Do they need to do anything different?

Reply:
Made it do what they usually do

Reply:
Yep, same command is fine, just for consistency so that we have separate script for ci stuff